tools/dgeni/processors/categorizer.js

```diff
@@ -55,12 +58,25 @@ module.exports = function categorizer() {
    * outputs will be marked. Aliases for the inputs or outputs will be stored as well.
    **/
   function decoratePropertyDoc(propertyDoc) {
+    decoratePublicDoc(propertyDoc);
+
     propertyDoc.isDirectiveInput = isDirectiveInput(propertyDoc);
     propertyDoc.directiveInputAlias = getDirectiveInputAlias(propertyDoc);
 
     propertyDoc.isDirectiveOutput = isDirectiveOutput(propertyDoc);
     propertyDoc.directiveOutputAlias = getDirectiveOutputAlias(propertyDoc);
   }
+
+  /**
+   * Decorates public exposed docs. Creates a property with CSS classes that will be 
+   * added to the template.
+   **/
+  function decoratePublicDoc(doc) {
+    // Specific classes that will can added to the Dgeni doc template.
+    doc.docClasses = [
```

Comment:
This step shouldn't really be concerned with css classes; it should just mark up the doc object with appropriate properties. Then, in the template we can use a nunjucks expression to apply the class